### ISSUE TITLE
Improvements to template repo

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pds-template-repo-python#13]
- [ ] Add a template "Hello World"-like doc source page so the sphinx docs can be built
- [ ] Create a Github Pages branch to push the docs to as part of CI/CD
- [ ] Add 1 template python test so python tests can successfully run via CI/CD (e.g. `sys.exit(1)`)

**Comment by MJJoyce** [comment]
Thoughts on adding a default code formatter (e.g., [black](https://github.com/psf/black)) and linter (e.g., [flake8](https://flake8.pycqa.org/en/latest/))?

I've had good luck using those with minor configuration changes to get automatic code formatting / linting, along with test suite execution, etc., with git hooks. I like to use [pre-commit](https://pre-commit.com/) since it mildly simplifies managing the installation of stuff.

**Comment by jordanpadams** [comment]
@MJJoyce I like the idea, but I'll leave it to @tloubrieu-jpl and @nutjob4life to make the call

**Comment by MJJoyce** [comment]
> * [ ]  Add a template "Hello World"-like doc source page so the sphinx docs can be built via CI/CD

A default index.html file is built as part of the current docs build. Are we looking for something more complicated than that?

> * [ ]  Add CONTRIBUTOR guide - [github says some things here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file) but open to something apache-like
> * [ ]  Add CODE_OF_CONDUCT - [example from some SMD folks](https://github.com/spacetelescope/style-guides/blob/master/CODE_OF_CONDUCT.md), but open to something apache-like

Potential solution to cover these in all of our repositories here:
https://github.com/nasa-pds/pds-github-util/issues/15#issuecomment-885930379

Not sure how folks feel about that.

**Comment by MJJoyce** [comment]
> * [ ]  Update stable_cicd.yaml to use latest roundup (e.g. triggered by `release/*` branch)

I think this was covered by @nutjob4life with 68e433f369f923e21bb21902046b99cbbae471c2? Sean can you confirm?

**Comment by nutjob4life** [comment]
> thoughts on adding a default code formatter (e.g., black) and linter (e.g., flake8)?

So long as we can have 120 columns I'm all for it. But let's make it part of the PDS Style Guide epic. I think keeping _this_ ticket a bit more tightly focused means we'll be able to get it done with the limited time you (and me) can devote to PDS work. 

> I think this was covered by @nutjob4life with 68e433f? Sean can you confirm?

Confirmed!

**Comment by MJJoyce** [comment]
> * [x]  Add a template "Hello World"-like doc source page so the sphinx docs can be built via CI/CD

Calling the above done since our docs build outputs an index file that (IMO) covers this.

> * [ ]  Add CONTRIBUTOR guide - [github says some things here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file) but open to something apache-like
> * [ ]  Add CODE_OF_CONDUCT - [example from some SMD folks](https://github.com/spacetelescope/style-guides/blob/master/CODE_OF_CONDUCT.md), but open to something apache-like

@nutjob4life / @jordanpadams / @tloubrieu-jpl, thoughts on the potential solution here:
https://github.com/NASA-PDS/pds-github-util/issues/15#issuecomment-885930379

TLDR: Make `nasa-pds/.github`, add the default versions of these files, and Github will allegedly make sure they show up in every repo that doesn't specify one. Seems like a painless way to make sure we always have these files around.

**Comment by MJJoyce** [comment]
Code of Conduct and Contributor guides are being handled in:
https://github.com/NASA-PDS/.github
